index.html

```diff
@@ -1138,7 +1139,8 @@ <h3>Data delivery</h3>
               Let |record:PressureRecord| be a new {{PressureRecord}} object with its
               {{PressureRecord/[[Source]]}} set to |source|,
               {{PressureRecord/[[State]]}} set to |state|
-              and {{PressureRecord/[[Time]]}} set to |timestamp|.
+              and {{PressureRecord/[[Time]]}} set to [=relative high resolution time=] based on |timestamp|
+              and a global.
```

Comment:
You can't say "a global" here, you need to pass an actual global (`relevantGlobal`, in this case) instead.

And, in fact, you always have the same global, so you can set this value outside the loop rather than having to recalculate it N times.

Reply:
Not sure if my solution is acceptable. 
We already have relevantGlobal set, so no need to set a new one.

Reply:
What I'm saying is that instead of

- collect data
- set `|timestamp|`
- for `|observer|` in `|relevantGlobal|`'s registered observer list
  - calculate the same value based on `timestamp` and `relevantGlobal`
  - set `observer.[[Time]]` to the value above

you do

- collect data
- set `|timestamp|`
- calculate value based on `timestamp` and `relevantGlobal` once
- for `|observer|` in `|relevantGlobal|`'s registered observer list
  - set `observer.[[Time]]` to the value above